### ISSUE TITLE
I'm not losing XP when I die

### ISSUE THREAD

**Issue by ASusuwatari** [AllTheMods/ATM-6#1783]
I'm not losing XP upon death and I feel like this defeats the challenge of a lot of the gameplay.  Is there a way to turn back on losing XP?

**Comment by jvfd3** [comment]
It is probably due to the tombstone mod.

I read the configs and found this:
"
	#Knowledge of Death loss why dying [0..500|default:0]
	#Range: 0 ~ 500
	knowledge_loss = 0
"

That's probably what you are looking for.

**Comment by ASusuwatari** [comment]
okay thanks! I'll check out if this works.
